curriculum/challenges/english/05-apis-and-microservices/mongodb-and-mongoose/create-a-model.english.md

```diff
@@ -6,32 +6,38 @@ challengeType: 2
 
 ## Description
 <section id='description'>
+<b>C</b>RUD Part I - CREATE
+
 First of all we need a Schema. Each schema maps to a MongoDB collection. It defines the shape of the documents within that collection.
 Schemas are building block for Models. They can be nested to create complex models, but in this case we’ll keep things simple.
 A model allows you to create instances of your objects, called documents.
-Create a person having this prototype :
-<code>- Person Prototype -</code>
-<code>--------------------</code>
-<code>name : string [required]</code>
-<code>age :  number</code>
-<code>favoriteFoods : array of strings (*) </code>
-Use the mongoose basic schema types. If you want you can also add
-more fields, use simple validators like required or unique,
-and set default values. See the <a href='http://mongoosejs.com/docs/guide.html'>mongoose docs</a>.
-[C]RUD Part I - CREATE
-Note: Glitch is a real server, and in real servers the interactions with the db happen in handler functions. These function are executed when some event happens (e.g. someone hits an endpoint on your API). We’ll follow the same approach in these exercises. The done() function is a callback that tells us that we can proceed after completing an asynchronous operation such as inserting, searching, updating or deleting. It’s following the Node convention and should be called as done(null, data) on success, or done(err) on error.
-Warning - When interacting with remote services, errors may occur !
-<code>/* Example */</code>
-<code>var someFunc = function(done) {</code>
-<code>&nbsp;&nbsp;//... do something (risky) ...</code>
-<code>&nbsp;&nbsp;if(error) return done(error);</code>
-<code>&nbsp;&nbsp;done(null, result);</code>
-<code>};</code>
+
+<strong>Note:</strong> Glitch is a real server, and in real servers the interactions with the db happen in handler functions. These function are executed when some event happens (e.g. someone hits an endpoint on your API). We’ll follow the same approach in these exercises. The <code>done()</code> function is a callback that tells us that we can proceed after completing an asynchronous operation such as inserting, searching, updating or deleting. It’s following the Node convention and should be called as <code>done(null, data)</code> on success, or <code>done(err)</code> on error.
```

Comment:
```suggestion
Glitch is a real server, and in real servers the interactions with the db happen in handler functions. These function are executed when some event happens (e.g. someone hits an endpoint on your API). We’ll follow the same approach in these exercises. The <code>done()</code> function is a callback that tells us that we can proceed after completing an asynchronous operation such as inserting, searching, updating or deleting. It’s following the Node convention and should be called as <code>done(null, data)</code> on success, or <code>done(err)</code> on error.
```
This is my opinion here, I like it better since it doesn't really seem to be a note - but it can be left alone if you want @RandellDawson.